src/form-generator/form-handler-api-web-forms.ts

```diff
@@ -0,0 +1,118 @@
+import { HtmlTagName } from "../html-tag-name";
+import { FormSuccessMessage } from "../form-element-creator/form-success-message";
+import { FormRequiredFieldsParagraph } from "../form-element-creator/form-required-fields-paragraph";
+import { DivCreatorWithClassName } from "../form-element-creator/div-creator";
+import { classNames } from "../config/class-name";
+import { config } from "../config/config-attributes";
+import { ButtonCreation } from "../form-element-creator/button-creation";
+
+export class FormHandlerApiWebForms {
+  private formElementCreation: FormElementCreation;
+  private formValidator: FormValidator;
+  private formSuccessMessage: FormSuccessMessage;
+  private formRequiredFieldsParagraph: FormRequiredFieldsParagraph;
+
+  constructor() {
+    this.formElementCreation = new FormElementCreation();
+    this.formValidator = new FormValidator();
+    this.formSuccessMessage = new FormSuccessMessage();
+    this.formRequiredFieldsParagraph = new FormRequiredFieldsParagraph();
+  }
+
+  createFormElement = async (): Promise<HTMLDivElement> => {
+    const container = new DivCreatorWithClassName().createDiv(
+      classNames.container
+    );
+    const title = new DivCreatorWithClassName().createDiv(
+      classNames.titleOneStep
+    );
+    title.textContent = config.titleOneStep.textContent;
+
+    const form = new HtmlTagName().formCreator();
+    form.id = "contact-form";
+
+    const elements = [...config.selects, ...config.fields];
+
+    for (const element of elements) {
+      if (element.name !== "Password" && element.name !== "Confirm Password") {
+        await this.formElementCreation.create(form, element);
+      }
+    }
+
+    const button = this.createSubmitButton(container);
+    form.appendChild(button);
+
+    container.appendChild(title);
+    container.appendChild(form);
+
+    const requiredParagraph =
+      this.formRequiredFieldsParagraph.createParagraphRequiredFields();
+    container.appendChild(requiredParagraph);
+
+    return container;
+  };
+
+  createSubmitButton = (container: any): any => {
+    const buttonCreator = new ButtonCreation();
+    const successMessage = this.formSuccessMessage.showSuccessMessage();
+      const clickHandler = async () => {
+      if (this.validateForm()) {
+        try {
+          const formData = new FormData(
+            document.querySelector("#contact-form") as HTMLFormElement
+          );
+          console.log(formData);
+          const formObject: { [key: string]: any } = {};
+          formData.forEach((value, key) => {
+            formObject[key] = value;
+          });
+
+          const response = await fetch("https://api.web3forms.com/submit", {
+            method: "POST",
+            headers: {
+              "Content-Type": "application/json",
+              Accept: "application/json",
+            },
+            body: JSON.stringify({
+              ...formObject,
+              access_key: "YOUR_GENERATED_API_KEY",
```

Comment:
nie moze byc to wpisane z palca. Musi byc do zmiennej przypisane. Bo nikt tego nie ustawi